doc/specs/vulkan/chapters/memory.txt

```diff
@@ -552,58 +552,57 @@ There must: be at least one memory type with the
 ename:VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT bit set in its
 pname:propertyFlags.
 
-The memory types are sorted according to a preorder which serves to aid in
-easily selecting an appropriate memory type.
-Given two memory types X and Y, the preorder defines [eq]#X {leq} Y# if:
-
-  * the memory property bits set for X are a strict subset of the memory
-    property bits set for Y.
-    Or,
-  * the memory property bits set for X are the same as the memory property
-    bits set for Y, and X uses a memory heap with greater or equal
-    performance (as determined in an implementation-specific manner).
-
-Memory types are ordered in the list such that X is assigned a lesser
-pname:memoryTypeIndex than Y if [eq]#(X {leq} Y) {land} {lnot} (Y {leq} X)#
-according to the preorder.
-Note that the list of all allowed memory property flag combinations above
-satisfies this preorder, but other orders would as well.
-The goal of this ordering is to enable applications to use a simple search
-loop in selecting the proper memory type, along the lines of:
+The array pname:memoryTypes must: be ordered in such a way that for each of
+its distinct memory types X and Y it holds that X occupies a lower index
+position than Y if:
+
+  * either the pname:propertyFlags of X (interpreted as a set of raised flags) is a
+    strict subset of the pname:propertyFlags of Y.
+  * or the pname:propertyFlags of X and Y are equal, and X belongs to a memory
+    heap with greater performance (as determined in an
+    implementation-specific manner).
+
+[NOTE]
+.Note
+====
+There might be more than one conformant way to order given set of memory
+types. Notice that the list of all allowed memory property flag
+combinations above is written in one such of several possible ways that do
+satisfy the prescribed order.
+====
+
+The purpose of such ordering is to enable applications to use a simple search
+loop to select the desired memory type along the lines of:
```

Comment:
nit: Perhaps this alternative sentence start?

```
This ordering is provided to enable applications to...
```

Reply:
nit: what about "... enables ...".

Reply:
As in "This ordering enables applications to..."?

Yes, less words meaning the same thing is always better :)